NDTensors/ext/NDTensorsMetalExt/adapt.jl

```diff
@@ -6,6 +6,8 @@ end
 
 # More general than the version in Metal.jl
 function Adapt.adapt_storage(arraytype::Type{<:MtlArray}, xs::AbstractArray)
-  arraytype_specified = specify_parameters(arraytype, get_parameters(xs))
+  arraytype_specified = specify_parameters(
+    arraytype, default_parameters(arraytype), parameters(xs)
+  )
```

Comment:
I'm confused about this usage of `specify_parameters`, what does it mean when both `default_parameters(arraytype)` and `parameters(xs)` are passed?

Reply:
This was a naming issue. I think we are still working on the naming below but `default_parameters` migrated to `parameter_names` though in principle we should be able to provide a list of positions.